portfolio/src/main/webapp/script.js

```diff
@@ -29,8 +29,40 @@ hideables.forEach(hideable => {
   h3.addEventListener('click', () => ul.classList.toggle("hidden"));
 });
 
-async function getGreeting() {
-    const response = await fetch("/data");
-    const quote = await response.text();
-    document.getElementById("quote-container").innerHTML = quote;
+async function getComments() {
+  const response = await fetch("/data");
+  const text = await response.text();
+  const comments = JSON.parse(text);
+
+  document.getElementById("loading-text").remove();
+  
+  const fatherDiv = document.getElementById("comments-container");
+
+  comments.forEach((comment) => {
+    //create div component for comment
+    const comDiv = document.createElement('div');
+    comDiv.classList.add("comment");
+
+    //create name header
+    const h4 = document.createElement('h4');
+    h4.classList.add('comment-name-date');
+    const node1 = document.createTextNode(`${comment.name}`);
```

Comment:
This should work, or do I miss something here?
```suggestion
    const node1 = document.createTextNode(comment.name);
```
Same comment for the following lines.